tensorrt_llm/_torch/pyexecutor/llm_request.py

```diff
@@ -49,6 +236,27 @@ def __init__(self, *args, **kwargs):
         self.py_draft_tokens = self.draft_tokens
         self.py_last_draft_tokens = None
 
+        self.py_return_log_probs = return_log_probs
+        self.py_return_context_logits = return_context_logits
+        self.py_return_generation_logits = return_generation_logits
+
+        # TODO: remove this when use DynamicDecodeOp in pytorch flow.
+        # currently, keep py_stop_words_list as python list, rather than tensor.
+        self.py_stop_words_list = stop_words_list
+
+        self.py_result = PyResult(self.py_prompt_len, self.py_max_new_tokens,
+                                  return_logits_device_memory, self.streaming,
+                                  return_log_probs, return_context_logits,
+                                  return_generation_logits)
+
+    def create_response(
+            self,
+            use_fast_logits=False,
+            mpi_world_rank=0) -> tensorrt_llm.bindings.executor.Response | None:
```

Comment:
The return type hint should be `LlmResponse | None`?

Reply:
Same as above.